lib/response.js

```diff
@@ -229,7 +241,28 @@ res.send = function send(body) {
     this.end();
   } else {
     // respond
-    this.end(chunk, encoding);
+    if (chunk && chunk.constructor && chunk.constructor.name === 'Blob') {
```

Comment:
this line is present 4 times, you could factorize it with 
```ts
const isChunkBlob = chunk && chunk.constructor && chunk.constructor.name === 'Blob'
```

Reply:
thank you it is resolved